app/assets/stylesheets/filter-form.scss

```diff
@@ -7,30 +7,18 @@
   @include govuk-clearfix;
 }
 
-.app-filter-layout__filter {
```

Comment:
This caused some issues on small width screens 
![Screenshot from 2024-04-16 15-05-13](https://github.com/DFE-Digital/itt-mentor-services/assets/11318084/e6b6faec-03c0-42fb-8cc0-ab74178b02ee)